python/version_details/version_details.py

```diff
@@ -586,7 +587,7 @@ def load_data(self, entity):
         self._requested_entity = entity
 
         # If we're pinned, then we don't allow loading new entities.
-        if self._pinned and self.current_entity:
+        if self._pinned and self._current_entity:
```

Comment:
@thebeeland These changes will be merged into master. But requesting review on dynamite-lib here. Also, these changes go along with https://github.com/shotgunsoftware/tk-multi-versiondetails/pull/22